pkg/controller/job/job_controller_test.go

```diff
@@ -80,6 +80,16 @@ func newJob(parallelism, completions int32) *batch.Job {
 	} else {
 		j.Spec.Parallelism = nil
 	}
+	if backoffLimit >= 0 {
+		j.Spec.BackoffLimit = &backoffLimit
+	} else {
+		j.Spec.BackoffLimit = nil
```

Comment:
This is generally not possible situation, since if everything goes through API it will get defaulted. So I'd just ignore this case.

Reply:
block replaced by
```j.Spec.BackoffLimit = &backoffLimit```